README.md

```diff
@@ -39,6 +39,12 @@ The site should now be live at `http://localhost:4000/capital-framework/`.
 $ grunt vendor
 ```
 
+**Deny/Allow connections.** If you are an employee of the CFPB, you may encounter the following message:
```

Comment:
I see no reason to make it CFPB-specific. I am fairly certain this dialog would pop up for anyone when they try to start a node server for the first time on an OS X installation.